debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/SourceInfo.java

```diff
@@ -8,6 +8,7 @@
 import java.time.Instant;
 
 import io.debezium.annotation.NotThreadSafe;
+import io.debezium.connector.oracle.xstream.LcrPosition;
```

Comment:
Is this actually used?

Reply:
I don't know hotestly. I did not touch XStream code, except a few necessary changes to make "switch" working